src/cryptonote_core/tx_pool.cpp

```diff
@@ -985,7 +987,7 @@ namespace cryptonote
       if (pit.second >= start_time)
         txids.push_back(pit.first);
     }
-    get_transactions_info(txids, added_txs, include_sensitive);
+    get_transactions_info(txids, added_txs, include_sensitive, limit_size);
     if (added_txs.size() > max_tx_count)
     {
       remaining_added_txids.reserve(added_txs.size() - max_tx_count);
```

Comment:
This `for` loop adding the remaining TXIDs needs updating to catch when txs were skipped because of the size limit. This logic won't do it correctly if any txs are skipped.

Reply:
Here's a commit that implements the logic for correctly handling skipped txs for this PR: https://github.com/jeffro256/monero/commit/93b35b1ecd1bc8a5d2ec8d22cfde15047a0ab526